src/libraries/System.Security.Cryptography.Cng/src/System/Security/Cryptography/AesCng.cs

```diff
@@ -201,8 +199,6 @@ public override void GenerateIV()
             int feedbackSizeInBits,
             out int bytesWritten)
         {
-            ValidateCFBFeedbackSize(feedbackSizeInBits);
```

Comment:
On the one hand, this had good call stack symmetry with the non-Windows implementations.  On the other hand it's the same exception from the same public member, just verified a little later.  So... it's fine.

Merely commenting because I had the thought.